lib/srfi/35/internal.scm

```diff
@@ -0,0 +1,206 @@
+(define-record-type Simple-Condition
```

Comment:
I have to think about this, but long term it may be better to unify the native exceptions with Simple-Condition.